tools/cloudera-parcel-gcsconnector/create_parcel.sh

```diff
@@ -176,7 +176,7 @@ function main() {
     graceful_exit
   fi
 
-  cp gcs-connector-hadoop2-latest.jar ${PARCEL_FULLNAME}/lib/hadoop/lib/
+  cp * ${PARCEL_FULLNAME}/lib/hadoop/lib/
```

Comment:
JSON key file is required for clusters that do not live on GCP. maybe only cp the gcs-connector-hadoop2-latest.jar and *.json instead of "*". These files will be replicated to every node of the cluster.